src/components/reusable/SearchBar.js

```diff
@@ -16,27 +20,12 @@ const Searchbar = ({ selectItem, getSearchResults }) => {
   // Which item in the drop down the user has selected
   const [highlightedIndex, setHighlightedIndex] = useState(-1);
 
-  // The search results to be displayed in the dropdown
-  const [searchResults, setSearchResults] = useState([]);
-
   const searchRef = useRef(null);
 
   // Handles traversing and choosing dropdown options
   const handleKey = (event) => {
     if (event.key === "Enter") {
-      // If an index has been highlighted, should navigate on enter
```

Comment:
This needs to still check if a `highlightedIndex` has been selected. Otherwise, any time the user presses enter then they will navigate to a new page and the page will not be for a valid ticker.

Reply:
done